app/services/service_listeners/publishing_api_pusher.rb

```diff
@@ -44,22 +44,17 @@ def handle_html_attachments(event)
       PublishingApiHtmlAttachmentsWorker.perform_async(edition.id, event)
     end
 
-    # If the previous edition had extra translations, redirect them to the :en locale.
-    # Unmigrated formats as of now (02/11/2016) do not work with this, as the redirects
```

Comment:
I don't see why they wouldn't just be republished and fixed like everything else. I'm going to run through this on integration today so I'll check and remove/update the comment as required.